src/fidesops/service/privacy_request/request_runner_service.py

```diff
@@ -25,79 +29,89 @@
 from fidesops.util.async_util import run_async
 from fidesops.util.cache import FidesopsRedis
 
+logger = logging.getLogger(__name__)
+
+_scheduler = None
+
+
+def get_scheduler() -> BackgroundScheduler:
+    """Returns a BackgroundScheduler as a singleton"""
+    global _scheduler  # pylint: disable=W0603
+    if _scheduler is None:
+        _scheduler = BackgroundScheduler()
+        _scheduler.start()
+    return _scheduler
+
 
 class PrivacyRequestRunner:
     """The class responsible for dispatching PrivacyRequests into the execution layer"""
 
     def __init__(
         self,
         cache: FidesopsRedis,
-        db: Session,
         privacy_request: PrivacyRequest,
     ):
         self.cache = cache
-        self.db = db
         self.privacy_request = privacy_request
 
-    def run_webhooks(
-        self, webhook_cls: WebhookTypes, after_webhook: Optional[WebhookTypes] = None
+    @staticmethod
+    def run_webhooks_and_report_status(
```

Comment:
As a general comment, would it make more sense to run the webhooks as a step in the task runner as some kind of pre-traversal task? That way the BackgroundScheduler would not be necessary.

Reply:
Ah, thanks. I do think that if we're going to have a background scheduler it would be a nice refactor to  define it in its own file. Right now we're also using this in `tasks.initiate_scheduled_request_intake` which is being called by `main`, so it's not limited to the privacy request.

Reply:
it was in its own file originally! I moved here for circular import reasons, it's tricky because `initiate_scheduled_request_intake` calls the privacy request runner, i'll see if i can figure out something better.

Reply:
ok I've moved into its own file, plus some adjustments to not instantiate on demand!